src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Tests/AmqpConverterTests.cs

```diff
@@ -139,7 +133,6 @@ void SB_message_with_no_TTL_results_in_empty_Ampq_TTL()
         }
 
         [Fact]
-        [LiveTest]
```

Comment:
I assume we removed this because it doesn't hit the live server, right?

Reply:
Correct.   There were a bunch of things that were bound to `TestUtility` which had dependencies on the environment variable due to static initialization.   I tweaked the approach for initialization and was able to reclassify ~15 tests that weren't actually `Live`.